test_app/scripts/bootstrap.sh

```diff
@@ -1,12 +1,34 @@
+#!/usr/bin/env bash
 
-rm -rf test_app/tests/sqllite_dbs/*.sqlite3
+set -e
 
-python manage.py migrate
+# Always make sure we kill the database when we exit
+function cleanup {
+    echo "Killing dab_postgres container"
+    docker kill dab_postgres
+}
 
-DJANGO_SUPERUSER_PASSWORD=password DJANGO_SUPERUSER_USERNAME=admin DJANGO_SUPERUSER_EMAIL=admin@stuff.invalid python manage.py createsuperuser --noinput
+trap cleanup EXIT
 
-python manage.py authenticators --initialize
+make postgres
 
-python manage.py create_demo_data
+for i in {1..5}; do
```

Comment:
I got to 3/5 for this, might want to bump this to 10 or something.

Reply:
will do